File: bukkit/src/main/java/io/turbo/clientblockcommon/ClientBlock.java
package io.turbo.clientblockcommon;

import io.turbo.clientblockcommon.listeners.PlayerListener;
import lombok.Getter;
import org.bukkit.Bukkit;
import org.bukkit.plugin.java.JavaPlugin;

public final class ClientBlock extends JavaPlugin {
   @Getter private static ClientBlock instance;

    public ClientBlock() {
        instance = this;
    }

    @Override
    public void onEnable() {
      getLogger().info("Enabled ClientBlock (common)");
      Bukkit.getPluginManager().registerEvents(new PlayerListener(), this);
    }
}

File: common-kotlin/src/main/kotlin/io/turbo/clientblockkotlin/ClientBlock.kt
package io.turbo.clientblockkotlin

import io.turbo.clientblockkotlin.listeners.PlayerListener
import org.bukkit.Bukkit
import org.bukkit.plugin.java.JavaPlugin

class ClientBlock : JavaPlugin() {


    override fun onEnable() {
        instance = this
        logger.info("Enabled ClientBlock (kotlin)")
        Bukkit.getPluginManager().registerEvents(PlayerListener(), this)
    }

    companion object {
        var instance: ClientBlock? = null

        fun i() = instance
    }
}
